packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -20,6 +20,11 @@ import {
 import { calculateServiceFee, getMaxRounds } from '@wallet-utils/coinjoinUtils';
 import { CoinjoinCustomStrategy } from './CoinjoinCustomStrategy';
 import { CoinjoinDefaultStrategy, CoinJoinStrategy } from './CoinjoinDefaultStrategy';
+import {
+    Feature,
```

Comment:
Maybe if you call it 
```suggestion
    FeatureFlag,
```
It would be more obvious what is it?

Reply:
I've tried to subconsciously avoid the term, but I think you're right.

Reply:
Started to renaming, but then again realized, that `Feature` name is more accurate. It is e.g. Feature.coinjoin, value is `'coinjoin'` – feature domain/key. It's not an feature flag. Feature flag with value true/false is what we get when we use `selectIsFeatureEnabled(state, Feature.coinjoin)`... So FeatureFlag can be more confusing at the end. Especially if we add some extra param to the feature config other than just `flag` and start using that as a config.